fe/fe-core/src/main/java/com/starrocks/server/StorageVolumeMgr.java

```diff
@@ -377,12 +378,25 @@ public void load(SRMetaBlockReader reader)
 
     @Override
     public void gsonPostProcess() throws IOException {
-        for (Map.Entry<String, Set<Long>> entry : storageVolumeToDbs.entrySet()) {
+        // If user upgrades from 3.0 and the createTableInfo and createDbInfo is replayed,
+        // the image will look like: "svToDbs":{"null":[12288,81921,49154,65541,20485]}, "svToTables":{"null":[12288]}
+        // The mapping null to dbs and tables should be removed. These code can be removed when 3.0 is not supported.
+        for (Iterator<Map.Entry<String, Set<Long>>> it = storageVolumeToDbs.entrySet().iterator(); it.hasNext();) {
```

Comment:
looks like Iterator is not necessary?

if want to change the `storageVolumeToDbs`, can just remove it directly before the for loop.

Reply:
done